core/templates/dev/head/pages/exploration_player/AudioTranslationManagerService.js

```diff
@@ -153,6 +153,9 @@ oppia.factory('AudioTranslationManagerService', [
       getSpeechSynthesisLanguageCode: function() {
         return LanguageUtilService.getAutogeneratedAudioLanguage(
           _explorationLanguageCode).speechSynthesisCode;
+      },
+      isCurrentAudioLanguageCodeSet : function(){
+        return (_currentAudioLanguageCode != null);
```

Comment:
Should be !==

Reply:
Ok ill change that.

Reply:
I removed this function as I changed the check to the array length of languagesInExploration.